app/lib/features/pins/pages/pin_page.dart

```diff
@@ -83,152 +87,110 @@ class PinPage extends ConsumerWidget {
               ],
             ),
           ),
-        ]);
-      }
-    } else {
-      actions.add(
-        PopupMenuItem(
-          onTap: () => showAdaptiveDialog(
-            context: context,
-            builder: (ctx) => ReportContentWidget(
-              title: 'Report this Pin',
-              description:
-                  'Report this content to your homeserver administrator. Please note that your administrator won\'t be able to read or view files in encrypted spaces.',
-              eventId: pinId,
-              roomId: pin.roomIdStr(),
-              senderId: pin.sender().toString(),
-              isSpace: true,
-            ),
-          ),
-          child: Row(
-            children: <Widget>[
-              Icon(
-                Atlas.warning_thin,
-                color: Theme.of(context).colorScheme.error,
+          PopupMenuItem(
```

Comment:
I am confused: why did you drop the `else`? Doesn't that mean that other people can't report this pin anymore?  But anyone should be able to report a pin...

Reply:
Ahh this should not be this way, must have overlooked on it. Thanks for pointing it out!